pystorm/component.py

```diff
@@ -232,12 +237,11 @@ def _setup_component(self, storm_conf, context):
             backup_count = self.storm_conf.get('pystorm.log.backup_count',
                                                10)
             log_file = join(log_path,
-                            ('pystorm_{topology_name}_{component_name}'
-                             '_{task_id}_{pid}.log'
-                             .format(topology_name=self.topology_name,
+                            (log_file_name
+                             .format(d=dict(topology_name=self.topology_name,
```

Comment:
Rather than doing this and having the `{d[component_name]}` in the format string, it seems a bit cleaner to just do things the same as we had before (e.g., `{component_name} in the format strings and pass `component_name` as a keyword argument).  `str.format` does not care if you pass extra unused keyword arguments.